setup.py

```diff
@@ -74,6 +74,7 @@
         'paasta_tools/autoscale_all_services.py',
         'paasta_tools/check_marathon_services_replication.py',
         'paasta_tools/cleanup_marathon_jobs.py',
+        'paasta_tools/cleanup_maintenance.py',
```

Comment:
Can you use entrypoints instead? Can you can name the entrpoint `paasta_cleanup_maintenance`? I feel like we pollute the path every time we add a non-namespaced command like this.